### PR TITLE
Add Helm Events to ACR Schema for Event Grid

### PR DESCRIPTION
**Pull request by jaysterp**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by jaysterp** [comment]
//cc @djyou @sajayantony

File: specification/eventgrid/data-plane/Microsoft.ContainerRegistry/stable/2018-01-01/ContainerRegistry.json [hunk 1]
```diff
@@ -94,6 +134,40 @@
...
+                    "description": "The MIME type of the referenced object.",
+                    "type": "string"
+                },
+                "size": {
+                    "format": "int64",
+                    "description": "The number of bytes of the content.",
+                    "type": "integer"
+                },
+                "digest": {
+                    "description": "The digest of the helm chart.",
```
[the remark this answers is not in the document]
Comment (jaysterp) [reply specification/eventgrid/data-plane/Microsoft.ContainerRegistry/stable/2018-01-01/ContainerRegistry.json]: Yes, thank you. I removed the use of `helm chart` and replaced it with the more generic `artifact`.

**Review by djyou — APPROVED** [review]

File: specification/eventgrid/data-plane/Microsoft.ContainerRegistry/stable/2018-01-01/ContainerRegistry.json [hunk 2]
```diff
@@ -25,6 +25,24 @@
             ],
             "properties": {}
         },
+        "ContainerRegistryChartPushedEventData": {
```
Comment (sanjaiganesh) [inline specification/eventgrid/data-plane/Microsoft.ContainerRegistry/stable/2018-01-01/ContainerRegistry.json]: >Chart [](start = 26, length = 5)

referring to the other comment.. should this also be renamed to Artifact ?

Comment (djyou) [reply specification/eventgrid/data-plane/Microsoft.ContainerRegistry/stable/2018-01-01/ContainerRegistry.json]: `Chart` is one type of `Artifact` and there will be other event types for other event types that inherit the same `ContainerRegistryArtifactEventData`

Comment (sanjaiganesh) [reply specification/eventgrid/data-plane/Microsoft.ContainerRegistry/stable/2018-01-01/ContainerRegistry.json]: sounds good.

---
In reply to: [252465044](https://github.com/Azure/azure-rest-api-specs/pull/5109#discussion_r252465044) [](ancestors = 252465044)

**Review by sanjaiganesh — CHANGES_REQUESTED** [review]
:clock1:

**Review by sanjaiganesh — APPROVED** [review]
:shipit:

**Review by praries880 — APPROVED** [review]

**Pull request merged**